### PR TITLE
fixes: remove padding-bottom when user is logged out

### PR DESCRIPTION
**Pull request by SMahil**
when user is logged out, padding bottom should be zero to container

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -1,15 +1,27 @@
...
+})
+export class AppComponent implements OnInit {
   
-  constructor(auth: AuthenticationService){
+  constructor(private auth: AuthenticationService, private broadcaster: Broadcaster){
     //auth.isLoggedIn();
   }
-}
\ No newline at end of file
+  loggedIn: Boolean = false;
```
Comment (joshuawilson) [inline src/app/app.component.ts]: What are you trying to do here? How does this relate to fixing the padding for login?

Usually the app.component.ts file is for bootstrapping the entire application. So this kind of thing is not written here.

Comment (SMahil) [inline src/app/app.component.ts]: [class.containerPadd-nologin] = "!loggedIn" I am toggling the state to know whether user is logged in or not. If not then removing the padding -bottom

Comment (sanbornsen) [inline src/app/app.component.ts]: First of all, this padding was occurring in the list page, we should not add any logical customization over the main router.

Comment (SMahil) [inline src/app/app.component.ts]: padding is being changed thats why this issue occurs again

**Comment by joshuawilson** [comment]
@SMahil this needs to rebase too.

File: src/app/work-item/work-item-list/work-item-list.component.scss [hunk 2]
```diff
@@ -7,41 +7,65 @@
...
-  background-color: $color-pf-black-300;
-  justify-content: space-between;
-  .list-type {
-    font-weight: 700;
-    margin-left: em(10);
+.detail-list-container{
+    padding-bottom: em(65);
+    height: 100%;
+    overflow: hidden;
+  .list-action {
```
Comment (sanbornsen) [inline src/app/work-item/work-item-list/work-item-list.component.scss]: Indentation looks incorrect here.

**Pull request merged**